tensorflow_addons/image/transform_ops_test.py

```diff
@@ -121,7 +121,7 @@ def test_transform_data_types(self):
         for dtype in _DTYPES:
             image = tf.constant([[1, 2], [3, 4]], dtype=dtype)
             self.assertAllEqual(
-                np.array([[4, 4], [4, 4]]).astype(dtype.as_numpy_dtype()),
+                np.array([[4, 4], [4, 4]]).astype(dtype.as_numpy_dtype(0)),
```

Comment:
I wonder if using `dtype.as_numpy_dtype` is enough, which yields a `numpy.dtype` like `tf.int32` -> `np.int32`.

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/optimizer_v2/adagrad_test.py#L77